CODEOWNERS

```diff
@@ -1185,6 +1185,8 @@ build.json @home-assistant/supervisor
 /tests/components/yeelight/ @zewelor @shenxn @starkillerOG @alexyao2015
 /homeassistant/components/yeelightsunflower/ @lindsaymarkward
 /homeassistant/components/yi/ @bachya
+/homeassistant/components/yolink/ @YoSmart-Inc
+/tests/components/yolink/ @YoSmart-Inc
```

Comment:
Is `@YoSmart-Inc` your company account and is it active? Otherwise, would it make sense to add `@matrixd2` to the codeowners list as well?